issue.go

```diff
@@ -1512,3 +1512,27 @@ func (s *IssueService) AddRemoteLinkWithContext(ctx context.Context, issueID str
 func (s *IssueService) AddRemoteLink(issueID string, remotelink *RemoteLink) (*RemoteLink, *Response, error) {
 	return s.AddRemoteLinkWithContext(context.Background(), issueID, remotelink)
 }
+
+//Get all issues type with context
+func (s *IssueService) GetIssueWithContext(ctx context.Context) ([]IssueType, *Response, error) {
```

Comment:
This function receives all IssueTypes. Would it make sense to rename the function here?

```suggestion
func (s *IssueService) GetIssueTypesWithContext(ctx context.Context) ([]IssueType, *Response, error) {
```